### PR TITLE
log sanitation on output of jobs

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR...

Fixes: #8853

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [x] Trivial

### CODE REVIEW

File: services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java [hunk 1]
```diff
@@ -3106,17 +3106,16 @@ protected String parseCifsMountOptions(URI uri) {
             extraOpts.append(name + "=" + nvp.getValue() + ",");
         }
 
-        if (logger.isDebugEnabled()) {
-            logger.error("extraOpts now " + extraOpts);
-        }
+        String extraOptions = extraOpts.toString();
+        logger.error("extraOpts now ", ()->StringUtils.cleanString(extraOptions));
```
Comment (sureshanaparti) [inline services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java:3110]: debug log?

Comment (DaanHoogland) [reply services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java:3110]: hm, decisions, decisions, decisions, ... ;)

maybe even trace, I just copied the original call.

Comment (hsato03) [reply services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java:3110]: Agree with @sureshanaparti. IMO, debug makes more sense in this case.

Comment (DaanHoogland) [reply services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java:3110]:
```suggestion
        logger.debug("extraOpts now ", ()->StringUtils.cleanString(extraOptions));
```

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test alma9 kvm-alma9

**Review by hsato03 — APPROVED** [review]
LGTM

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Review by sureshanaparti — APPROVED** [review]
LGTM

**Pull request merged**